docs/building-and-flashing/build.md

```diff
@@ -179,7 +179,7 @@ The supported way to flash when developping for the Crazyflie is to use the Craz
 
 #### Prerequisites
 * A Crazyradio with drivers installed
-* [Crazyflie Client installed](https://github.com/bitcraze/crazyflie-clients-python) with Python's pip (so not by Snap (Ubuntu) or the .exe (Windows))
+* [Crazyflie Client](https://github.com/bitcraze/crazyflie-clients-python) installed in a Python virtual environment, either via pip (not using Snap on Ubuntu or .exe on Windows) or by cloning from source.
```

Comment:
```suggestion
* [Crazyflie Client](https://github.com/bitcraze/crazyflie-clients-python) installed in a Python virtual environment, either via pip or by cloning from source.
```

The snaps are extremely outdated, I didn't even know we distributed .exe files, and probably no one uses them anymore anyway. The sentence is cleaner and conveys the same essential information without these obsolete references.